app/assets/stylesheets/custom.css.scss

```diff
@@ -71,3 +71,28 @@ h4 {
   text-align: center;
   margin-bottom: 20px;
 }
+
+.item-create {
+  background: rgba(43,58,223,0.2);
+  width: 500px;
+  border: 1em;
+  border-color: black;
+  border-style: solid;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.item-update {
+  background: rgba(43,58,223,0.2);
+  width: 500px;
```

Comment:
Properties should be ordered background, border, border-color, border-style, margin-left, margin-right, width